app/templates/admin/content/events.hbs

```diff
@@ -0,0 +1,157 @@
+                      </div>
+                    </div>
+                  </div>
+                </td>
+              </tr>
+            {{/each}}
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div class="six wide column">
+      <div class="ui center aligned fluid segment">
+        <div class="ui grid container">
+          <div class="middle aligned content eight wide column left aligned">
+            <h4 class="ui left aligned header">
+              {{t 'Event Topics'}}
+            </h4>
+          </div>
+          <div class="eight wide column right aligned">
+            <button class="ui icon circular blue button {{if device.isMobile 'fluid'}}" {{action 'openNewEventTopicModal'}}>
+              <i class="icon plus"></i>
+            </button>
+          </div>
+        </div>        
+        <div class="ui divider"></div>
+        <table class="ui celled table">
+          <tbody>
+            {{#each model.eventTopics as |eventTopic|}}
+              <tr>
+                <td>
+                  <div class="ui grid">
+                    <div class="middle aligned content eight wide column left aligned">
+                      {{eventTopic.name}}
+                    </div>
+                    <div class="eight wide column right aligned">
+                      <div class="ui horizontal compact basic buttons">
+                        {{#ui-popup content=(t 'Edit') class='ui icon button' click=(action 'openNewEventTopicModal' eventTopic)}}
+                          <i class="edit icon"></i>
+                        {{/ui-popup}}
+                        {{#ui-popup content=(t 'Delete') class='ui icon button' click=(action (confirm (t 'Are you sure you would like to delete this event topic?') (action 'deleteEventTopic' eventTopic)))}}
+                          <i class="trash outline icon"></i>
+                        {{/ui-popup}}
+                      </div>
+                    </div>
+                  </div>                  
+                </td>
+              </tr>
+            {{/each}}
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div class="five wide column">
+      <div class="ui center aligned fluid segment">
+        <div class="ui grid container">
+          <div class="middle aligned content eight wide column left aligned">
+            <h4 class="ui left aligned header">
+              {{t 'Event Subtopics'}}
+            </h4>
+          </div>
+          <div class="eight wide column right aligned">
+            <button class="ui icon circular blue button {{if device.isMobile 'fluid'}}" {{action 'openNewEventSubTopicModal'}}>
+              <i class="icon plus"></i>
+            </button>
+          </div>
+        </div>   
+        <div class="ui divider"></div>
+        <div class="field">
+          <label>{{t 'Event Topic'}}</label>
+          {{#ui-dropdown class='search selection' onChange=(action (mut model.topic)) as |execute mapper|}}
```

Comment:
here instead of `(action (mut model.topic))` make it something like `(action 'loadSubTopic')`.

Then define an action like, 

```js
async loadSubTopic(topic) {
  this.set('subTopics', await topic.get('subTopics');
}
``` 

Make sure to change the implementation in the new subtopic modal too. Also, ensure you have a loading indicator that comes up when the subtopics are loading.